src/components/Modeler.vue

```diff
@@ -629,6 +632,36 @@ export default {
         this.renderPaper();
       });
     },
+
+    getBoundaryEvents(process) {
+      return process.flowElements
+        ? process.flowElements.filter(({$type}) => $type === 'bpmn:BoundaryEvent')
+        : [];
```

Comment:
Not sure if I commented on this before 🤔 But I think `process.get('flowElements')` will always return an array, so this whole thing could just be:
```js
return process.get('flowElements').filter(({$type}) => $type === 'bpmn:BoundaryEvent');
```

Reply:
It is `Cannot read property 'filter' of undefined` when you load up a new process in processmaker. So that doesn't seem to be true.

Reply:
Done